client/lib/paths/login/index.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Internal dependencies
+ */
+import { addQueryArgs } from 'lib/url';
+import config, { isEnabled } from 'config';
+
+export function login( { redirectTo } ) {
+	const url = isEnabled( 'wp-login' ) ? '/login' : config( 'login_url' );
+
+	return redirectTo
+		? addQueryArgs( { redirect_to: redirectTo }, url )
```

Comment:
Yes, it does. See https://github.com/Automattic/wp-calypso/pull/13212/commits/7d67b179307bb5f41689c38746400bc7ee6a351a.